packages/editor/src/components/inserter/index.js

```diff
@@ -100,27 +100,19 @@ class Inserter extends Component {
 }
 
 export default compose( [
-	withSelect( ( select, { rootClientId, index } ) => {
+	withSelect( ( select, { rootClientId } ) => {
 		const {
 			getEditedPostAttribute,
 			getBlockInsertionPoint,
 			hasInserterItems,
 		} = select( 'core/editor' );
 
-		if ( rootClientId === undefined && index === undefined ) {
-			// Unless explicitly provided, the default insertion point provided
-			// by the store occurs immediately following the selected block.
-			// Otherwise, the default behavior for an undefined index is to
-			// append block to the end of the rootClientId context.
-			const insertionPoint = getBlockInsertionPoint();
-			( { rootClientId, index } = insertionPoint );
-		}
+		rootClientId = rootClientId || getBlockInsertionPoint().rootClientId;
 
 		return {
 			title: getEditedPostAttribute( 'title' ),
 			hasItems: hasInserterItems( rootClientId ),
-			rootClientId,
-			index,
+			rootClientId: rootClientId || getBlockInsertionPoint().rootClientId,
```

Comment:
Should this have been changed from `rootClientId,` ? There's redundancy here in that the same fallback behavior occurs a few lines above.